chisel/README.md

```diff
@@ -51,8 +51,8 @@ Chisel aims to improve upon soli, with native foundry integration by providing f
   - [x] Inspect storage vars
     - [ ] Inspect raw storage slots
   - [ ] Inspection verbosity configuration
-  - [ ] Enable / disable call trace printing
-    - [ ] Rip trace printing code from another module of foundry.
+  - [x] Enable / disable call trace printing
+    - [x] Rip trace printing code from another module of foundry.
```

Comment:
can we be sensitive about code duplication in this PR? I'd encourage looking for refactors in the tracing + debugging inspectors.

Reply:
Definitely- can you elaborate a bit? We're using the tracing inspector w/o any modifications, and the debugger inspector is disabled.

At the moment, the trace printing logic consists of stripped versions of the `decode_traces` and `show_traces` functions from scripting ([~70 lines](https://github.com/whitenois3/foundry/blob/feat/repl/chisel/src/dispatcher.rs#L311-L369)). I will need to add the `LocalTraceIdentifier` back in after the switch to using ethers' `Project` for compilation, though.